test/cli/test-fail/missing-resource/test.yaml

```diff
@@ -1,11 +1,16 @@
-name: test-simple
-policies:
+apiVersion: cli.kyverno.io/v1alpha1
+kind: PolicyTests
+metadata:
+ name: test-simple
+spec:
+ policies:
   - policy.yaml
-resources:
+ resources:
   - resources.yaml
-results:
+ results:
   - policy: disallow-latest-tag
     rule: validate-image-tag
     resource: missing
     kind: Pod
     result: pass
+   
```

Comment:
Not sure how this is happening - seems to be somewhitespace at the end of the file?

Reply:
I have made these changes now @MarcelMue.
Thank you!

Reply:
@MarcelMue please let me know if this is okay now. Thank you!